### ISSUE TITLE
How I know the searched people is the same as the people in the gallery?

### ISSUE THREAD

**Issue by Yue-Rain** [serend1p1ty/SeqNet#6]
a person may appear in serveral pictures,and the dataset has the unique person ID for every one?
because gallery and query has duplicate pics, and one pic may contain serveral persons, so you compute IOU between searched perosn box with groud truth to get positive case?
thank you!

**Comment by serend1p1ty** [comment]
1. All people are divided into labeled people and unlabeled ones. Each labeled people has a unique ID.
2. Gallery and query have no duplicate pics.
3. We extract the 256-dim normalized features for each person, and calculate the cosine similarity for them to judge whether two people are the same person.
You can find more details in the paper and [evaluation code](https://github.com/serend1p1ty/SeqNet/blob/master/eval_func.py).

**Comment by Yue-Rain** [comment]
After the function _compute_iou in the  https://github.com/serend1p1ty/SeqNet/blob/58471df02baacd338a53037e8856bc05b2be2229/eval_func.py#L246,the model think the j-th detection is correct,can we multiply sim score by 2 to highlight the j-th detection? I have encountered such a scenario which sim socre of the overall detection result is low,while the sim score of the wrong candidate is higher than that of the correct one.Rank1 is improved by doubling sim score of j-th detection.
`if _compute_iou(roi, gt) >= iou_thresh:
      label[j] = 1
      count_tp += 1
      break`
replaced with 
`if _compute_iou(roi, gt) >= iou_thresh:
      label[j] = 1
      sim[j] *= 2
      count_tp += 1
      break`
Another thing,could you upload models file and dataset to baidudisk?google drive can not be used.
Thank you!

**Comment by serend1p1ty** [comment]
Top-1 can be improved by multiplying the similarity of the j-th detection by 2 when you know that the j-th detection is ground truth, but it is a "cheating" behavior. In addition, multiplying by a larger number will further improve Top-1. For fair comparison, the evaluation rule **CAN NOT** be modified.

**Comment by Yue-Rain** [comment]
i want to finetune the second part of the seqnet model with custom data, while freezing the first part, detection.how can i do that? By setting the requires_grad attribute of the parameter of backbone and rpn of model is False?
`    for param in model.backbone.parameters:
`
`         param.requires_grad = False`
`    for parm in model.rpn.parameters:
`
`         param.requires_grad = False`
Thank you!
